src/map/treasure_pool.h

```diff
@@ -26,18 +26,34 @@
 
 #include <vector>
 
+enum TREASUREPOOLMANAGEMENT
+{
+    // Membership updated by party/alliance
+    TREASUREPOOL_MANAGED = 0,
+
+    // Membership updated by lua bindings or zone
+    TREASUREPOOL_UNMANAGED
+};
+
+// Update xi.treasurePool accordingly when making changes
 enum TREASUREPOOLTYPE
```

Comment:
While you're here, can you make this: `enum class TreasurePoolType : uint8`, and then the entries will be `Solo`, `Party`, etc.